### PR TITLE
Fix vcenter issues

### PR DESCRIPTION
**Pull request by naumvd95**
https://mirantis.jira.com/browse/FPL-2710
desc:
1) bad substitution with ":?" bash wrap
2) seems bad defining variables in vcenter reporter
3)fix wrong test_suite naming
see desc and comments in JIRA dashboard

### CODE REVIEW

File: servers/partner-ci/9.x/vcenter.yaml [hunk 1]
```diff
@@ -81,14 +81,14 @@
     skip_vote_failed: false
 
     testrail_project: "Mirantis OpenStack"
-    testrail_suite: "[9.x][PCE] vCenter"
+    testrail_suite: "[9.1][PCE] vCenter"
     testrail_suite_desc: "Suite description"
     testrail_milestone: "9.2"
-    testrail_plan: "$TREP_TESTRAIL_MILESTONE snapshot #${{SNAPSHOTS_ID}}"
+    testrail_plan: "${{TREP_TESTRAIL_MILESTONE}} snapshot #${{SNAPSHOTS_ID}}"
```
Comment (rkhozinov) [inline servers/partner-ci/9.x/vcenter.yaml]: SNAPSHOTS_ID and TREP_TESTRAIL_MILESTONE are parameters and will be used as parameters.
Jenkins job parse them as bash variables before job run. 
Do you check it on jenkins?

File: servers/partner-ci/9.x/vcenter.yaml [hunk 2]
```diff
@@ -166,7 +166,18 @@
           testgroup_value_type: choices
           testgroup_value: '{obj:testgroup}'
 
-      - '{reporter-name}'
+      - '{reporter-name}':
```
Comment (rkhozinov) [inline servers/partner-ci/9.x/vcenter.yaml]: Reporter should use global variables, we don't need duplicate pieces of the code

File: servers/partner-ci/builders/vcenter.test.sh [hunk 3]
```diff
@@ -62,7 +62,7 @@ if [[ $ISO_FILE == *"Mirantis"* ]]; then
   export FUEL_RELEASE=$(echo $ISO_FILE | cut -d- -f2 | tr -d '.iso')
 fi
 
-export ENV_NAME="${ENV_PREFIX}.${SNAPSHOTS_ID:?}"
```
Comment (rkhozinov) [inline servers/partner-ci/builders/vcenter.test.sh:65]: what's the problem?

File: servers/partner-ci/common.yaml [hunk 4]
```diff
@@ -875,8 +875,8 @@
           mask-password-params: true
     parameters:
       - string:
-          name: ISO_VERSION
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml:878]: Are you sure that it won't affect previous releases?

File: servers/partner-ci/builders/vcenter.test.sh (hunk 3 above)
Comment (rkhozinov) [reply servers/partner-ci/builders/vcenter.test.sh:65]: attach a trace

File: servers/partner-ci/9.x/vcenter.yaml (hunk 2 above)
Comment (naumvd95) [reply servers/partner-ci/9.x/vcenter.yaml]: i know , its just for testing

File: servers/partner-ci/builders/vcenter.test.sh (hunk 3 above)
Comment (naumvd95) [reply servers/partner-ci/builders/vcenter.test.sh:65]: repeat
SEE FULL DESCRIPTION IN JIRA tASK

File: servers/partner-ci/common.yaml (hunk 4 above)
Comment (naumvd95) [reply servers/partner-ci/common.yaml:878]: sorry its mistake

File: servers/partner-ci/builders/vcenter.test.sh (hunk 3 above)
Comment (rkhozinov) [reply servers/partner-ci/builders/vcenter.test.sh:65]: `line 65: SNAPSHOTS_ID: parameter null or not set ` 
`export SNAPSHOTS_ID= ` - it's the problem
```
+ [[ 610 == *\l\a\s\t\S\u\c\c\e\s\s\f\u\l\B\u\i\l\d* ]]
+ export SNAPSHOTS_ID= << it's the problem
+ SNAPSHOTS_ID=
+ '[' -f build.plugin_version ']'
++ grep PLUGIN_VERSION
++ cut -d= -f2
+ version=3.2.0
+ export DVS_PLUGIN_VERSION=3.2.0
+ DVS_PLUGIN_VERSION=3.2.0
+ '[' -z 116 ']'
+ '[' -f nosetest.xml ']'
+ sudo rm -rf 'logs/*'
+ [[ MirantisOpenStack-9.0.iso == *\M\i\r\a\n\t\i\s* ]]
++ echo MirantisOpenStack-9.0.iso
++ tr -d .iso
++ cut -d- -f2
+ export FUEL_RELEASE=90
+ FUEL_RELEASE=90
/tmp/hudson722437583800698906.sh: line 65: SNAPSHOTS_ID: parameter null or not set
```

File: servers/partner-ci/9.x/vcenter.yaml (hunk 2 above)
Comment (rkhozinov) [reply servers/partner-ci/9.x/vcenter.yaml]: ok

File: servers/partner-ci/builders/vcenter.test.sh [hunk 5]
```diff
@@ -28,8 +24,7 @@ if [ "${SNAPSHOTS_ID}" != "released" ]; then
 fi
 
 [[ $SNAPSHOTS_ID == *"lastSuccessfulBuild"* ]] && \
-  id=$(grep -Po '#\K[^ ]+' < snapshots.params)
-  export SNAPSHOTS_ID=$id
+  export SNAPSHOTS_ID=$(grep -Po '#\K[^ ]+' < snapshots.params)
```
Comment (naumvd95) [inline servers/partner-ci/builders/vcenter.test.sh:27]: ok

File: servers/partner-ci/builders/vcenter.test.sh [hunk 6]
```diff
@@ -124,8 +119,8 @@ setup_bridge() {
   sudo ip link set dev $bridge up
 
   if sudo iptables-save | grep $bridge | grep -i reject | grep -q FORWARD; then
-    iptables -D FORWARD -o $bridge -j REJECT --reject-with icmp-port-unreachable
-    iptables -D FORWARD -i $bridge -j REJECT --reject-with icmp-port-unreachable
+    sudo iptables -D FORWARD -o $bridge -j REJECT --reject-with icmp-port-unreachable
```
Comment (naumvd95) [inline servers/partner-ci/builders/vcenter.test.sh:98]: perfect

File: servers/partner-ci/9.x/vcenter.yaml (hunk 1 above)
Comment (rkhozinov) [inline servers/partner-ci/9.x/vcenter.yaml:87]: it's ok

**Review by rkhozinov — APPROVED** [review]

**Pull request merged**